### ISSUE TITLE
Doesn't work with latest AutoMapper version.

### ISSUE THREAD

**Issue by jrhiston** [TylerCarlson1/Automapper.Collection#13]
Tried installing with version 5.0.2 of AutoMapper. Seems to be failing due to inaccessibility issues, such as TypeHelper and TypeExtensions.

**Comment by TylerCarlson1** [comment]
Opps sorry missed this.  Should be fixed from #16 and version 2.0.1 has fix

**Comment by jrhiston** [comment]
Now works with version 5.0.2 of AutoMapper.
